website/templates/website/error_pages/404.html

```diff
@@ -1,9 +1,19 @@
 {% extends "website/base_template.html" %}
 
```

Comment:
since you are using `"{% static "` , you need to load the static. So you need to add `{% load staticfiles %}`